docs/contributor-guide/how-to/common-rule-scenarios.md

```diff
@@ -0,0 +1,205 @@
+# Implement common rule scenarios
+
+This page documents the most common scenarios encoutered when developing a rule
+for `sonarwhal`. If there's something that you want to do that is not
+documented here, please [open an issue][new issue] so we can help you.
+
+## Change feedback based on browser support
+
+Users can tell `sonarwhal` what browsers are important for them via the
+[`browserslist` property in `.sonarwhalrc`][browserconfiguration] or in
+the `package.json` file. You can have access to the list, and thus modify
+the feedback of your rule, via the property `context.targetedBrowsers`.
```

Comment:
Remove `,`.

Reply:
which one? I don't think there's any extra `,` here.


> and thus modify the feedback of your rule

clarifies only and can be removed so it should be "wrapped" in `,` (at least that's how it works in Spanish).